packages/engine-server/src/workspace/service.ts

```diff
@@ -1097,9 +1097,17 @@ export class WorkspaceService implements Disposable, IWorkspaceService {
     });
     const wsRoot = this.wsRoot;
     if (!vault.remote || vault.remote.type !== "git") {
-      throw new DendronError({ message: "cloning non-git vault" });
+      throw new DendronError({
+        message: "Internal error: cloning non-git vault",
+      });
+    }
+    let repoPath: string;
+    if (vault.selfContained) {
+      // vault2Path gives the path to the `notes` folder for self contained vaults
```

Comment:
1. vault2Path is used across the codebase. are there other places where this issue props up? 
2. since we have self contained vaults going forward, we should encapsulate this logic in a function (eg. vault2SCPath)

Reply:
> Are there other places where this issue props up?

I'm not sure, I never tried going through all the invocations. I'll add a task to skim over the invocations and see if any of them need to be updated.

> since we have self contained vaults going forward, we should encapsulate this logic in a function (eg. vault2SCPath)

Will do, sure.